app/structures/structures.go

```diff
@@ -1,3 +1,4 @@
+// Package structures provides item related structures
```

Comment:
I would say it is not very clear as I don't understand what it stands for when reading it. 
Actually the word structure at the end is not clear.
"item related" -> "item-related"

Reply:
maybe like this ? // Package structures provides item-related structs.